src/database/models/Launches.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  DataType,
+  Column,
+  Model,
+  Table,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+} from "sequelize-typescript"
+
+import { Site } from "@database/models/Site"
+import { User } from "@database/models/User"
+
+@Table({ tableName: "launches", paranoid: true })
+export class Launches extends Model {
+  @Column({
+    autoIncrement: true,
+    primaryKey: true,
+    allowNull: false,
+    type: DataType.BIGINT,
+  })
+  id!: number
+
+  @ForeignKey(() => User)
+  @Column
+  userId!: string
+
+  @ForeignKey(() => Site)
+  @Column
+  siteId!: string
```

Comment:
hmm, if we want to set this up so that sequelize retrieves the associated model, we might have to use [model association](https://github.com/sequelize/sequelize-typescript#model-association)

Reply:
@seaerchin hmm, wouldnt the `@ForeignKey(() => User)` already allow us to associate a relationship between launches and users? Exactly what am I missing ya?

Reply:
hmm i'm actually not sure here! i can't recall off the top of my head if it does so could i trouble you to test this?

Reply:
b806ef7